### PR TITLE
G0 5533 responsive work

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/invoice.component.html [hunk 1]
```diff
@@ -2,11 +2,61 @@
...
+                            <li role="menuitem">
+                                <a class="dropdown-item cp">pending</a>
+                            </li>
+                            <li role="menuitem">
+                                <a class="dropdown-item cp">settings</a>
+                            </li>
+                            <li role="menuitem">
+                                <a class="dropdown-item cp">template</a>
+                            </li>
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/invoice.component.html:49]: Remove blank space

File: apps/web-giddh/src/app/invoice/invoice.component.html [hunk 2]
```diff
@@ -2,11 +2,61 @@
...
+                            
+                        </ul>
+                </div>
 
                 <!--<invoice-page-dd (pageChanged)="goToRoute($event)" (selectVoucher)="voucherChange($event)"></invoice-page-dd>-->
-                <div class="" id="settingTab">
+                <div class="" id="setting-tab">
                     <ng-template [ngIf]="selectedVoucherType === 'debit note' || selectedVoucherType === 'credit note'">
 
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/invoice.component.html:59]: remove blank space

File: apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss [hunk 3]
```diff
@@ -383,7 +383,7 @@ span.close-modal.cp.pull-right {
     }
 }
 
-.changeStatus-dropdown #settingTab {
+.changeStatus-dropdown #setting-tab {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss:386]: Can we change changeStatus-dropdown to change-status-dropdown also?

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 4]
```diff
@@ -151,25 +144,45 @@
                                 Note
                             </button>
                         </div>
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:147]: Remove space

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 5]
```diff
@@ -151,25 +144,45 @@
                                 Note
                             </button>
                         </div>
+                        
                     </div>
+                    
                 </div>
             </div>
             <!-- Invoice E-way bill, export to excel for a single selected item along with
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:152]: Remove space

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 6]
```diff
@@ -151,25 +144,45 @@
                                 Note
                             </button>
                         </div>
+                        
                     </div>
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:149]: Remove space

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 7]
```diff
@@ -151,25 +144,45 @@
...
-                        <input type="text" placeholder="Search Invoice No." class=" form-control" #searchBox name="voucherNumberInput" [formControl]="voucherNumberInput" />
-                    </div>
-                </div>
-                <div class="col-sm-6 col-xs-6 full-width-on-mobile">
-                    <div class="input-container">
-                        <input type="text" placeholder="Search Customer Name" class=" form-control" #searchBox name="accountUniqueNameInput" [formControl]="accountUniqueNameInput" />
+        </div>
+        <!-- For mobile screen sizes -->
+       
+
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:157]: Remove blank space

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.scss [hunk 8]
```diff
@@ -1,5 +1,23 @@
...
+    width: 36px;
+    height: 36px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 18px;
+    border: 1px solid $gainsboro;
+    border-radius: 2px;
+    color:gray;
+    background: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/invoice/recurring/recurring.component.scss:17]: Use color var

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 9]
```diff
@@ -9352,35 +9402,19 @@ input.checkbox-custom[type="checkbox"] {
...
-    }
 }
 
 @media (max-width: 375px) {
-    #settingTab .tab-container .nav.nav-tabs,
-    .custom-tabs .nav.nav-tabs {
-        top: 78px;
-    }
-
+    // #setting-tab .tab-container .nav.nav-tabs,
```
Comment (sadik-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9414]: use CSS comment otherwise remove this code

Comment (sadik-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9414]: use CSS comment otherwise remove this code

**Pull request merged**